impeller/renderer/backend/vulkan/context_vk.cc

```diff
@@ -120,12 +126,14 @@ void ContextVK::Setup(Settings settings) {
     return;
   }
 
-  if (!settings.worker_task_runner) {
-    VALIDATION_LOG
-        << "Cannot set up a Vulkan context without a worker task runner.";
-    return;
-  }
-  worker_task_runner_ = settings.worker_task_runner;
+  raster_message_loop_ = fml::ConcurrentMessageLoop::Create(4u);
+#ifdef FML_OS_ANDROID
```

Comment:
IMO, we should isolate all the platform specific bits in at least in `//impeller/base` (in a later patch is fine).